log/tracing/localStorage.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+
+const { createNamespace } = require('continuation-local-storage')
```

Comment:
and no ';' here? Is there any reason?

Reply:
This module was initially written by a different team and they had a convention for using semicolons. I've added new code using our style, now it's mixed so I'm not entirely happy with it but I think it's possible we'll be able to remove all semicolons later on.